app/views/requests/_no_sunet_form.html.erb

```diff
@@ -2,28 +2,49 @@
   <div id="no-sunetid-form" aria-hidden="false">
     <%= f.fields_for :user, (current_request.user unless current_request.user && current_request.user.webauth_user?) || User.new do |uf| %>
       <% if f.object.requestable_with_library_id? %>
-        <div class='form-group'>
+        <div class="form-group <%= "has-error" if f.object.library_id_error? %>">
           <%= uf.label :library_id, 'Stanford Library ID', class: "control-label #{label_column_class} #{'required' unless f.object.requestable_by_all?}" %>
           <div class='<%= content_column_class %>'>
             <%= uf.text_field_without_bootstrap(
                   :library_id,
                   class: 'form-control',
+                  minlength: (10 unless f.object.requestable_by_all?),
+                  maxlength: 10,
+                  'aria-describedby' => 'library-id-help',
                   data: { behavior: 'single-user-field' },
                   autocomplete: 'off'
                 )
             %>
+            <% if f.object.library_id_error? %>
+              <div role="alert" id="library-id-help" class="alert alert-danger">
+                <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+                <%= f.object.errors[:library_id].join('. ') %>.<br/>
+                A Stanford Library ID has 10 digits. On some University ID cards, it has a prefix of <span class="suid-prefix" aria-label="8 0 6 0 9">80609</span>.
+              </div>
+            <% else %>
+              <p id="library-id-help" class="help-block">
```

Comment:
I forget -- do we support entering a proxy borrowers ID in this field?

Reply:
From what I understand, anybody who requests items w/ an ID that is not this 10 digit number does not do so through this request form (e.g. through workflows directly, or some other mechanism)

Reply:
@saseestone/@jvine may have more details

Reply:
Proxy borrower IDs should also be 10 digits. Pseudo-patron IDs may not be, but they can request through Workflows.